### DISCUSSION TITLE
getting mergeCells count value?

### DISCUSSION THREAD

**Discussion by markocam** [dfinke/ImportExcel#1493] [Q&A] [answered]
Hi,
using Import Excel is there a function to get the mergeCells count value from each worksheet?

**Comment by dfinke** [comment]
There is no specific function for that. Need to know more about this. It may be workable from the object model itself.

**Comment by markocam** [comment]
Hi Doug, thanks for getting back to me. 
So i found the value by changing the excel to a zip and extracting it. then in xl > worksheets open the worksheet  that has the merged cells sheet1.xml and if there is merged cells in the values and the list of merged cells is at the end of the xml, just not sure how to get to that count value

**Comment by markocam** [comment, marked as answer]
arr got it

$eFile = "$PSScriptroot\Test.XLSX"
$dTemp = Open-ExcelPackage $eFile

$ws = "sheet1"
$Sheet1 = $dTemp.Workbook.Worksheets["$ws"]
$c = $Sheet1.MergedCells.Count

Write-Host "merged cells count: $c"

Close-ExcelPackage $dTemp -NoSave

**Comment by dfinke** [comment]
Thanks for posting and using ImportExcel.

Curious how you use the count?

**Reply by markocam** [reply]
i just needed to find out if the worksheet has merged cells and if so how many, as there is a limitation in processing the excel documents in a eDocument analytics program so need to omit them from a search

**Reply by dfinke** [reply]
Sweet!
